r/tests/testthat/test-dplyr-funcs-conditional.R

```diff
@@ -192,6 +212,25 @@ test_that("case_when()", {
     tbl
   )
 
+  # with namespacing
+  compare_dplyr_binding(
+    .input %>%
+      transmute(cw = dplyr::case_when(chr %in% letters[1:3] ~ 1L) + 41L) %>%
+      collect(),
+    tbl
+  )
+
+  compare_dplyr_binding(
```

Comment:
Why do we need more than one test that you can call the namespaced version?

Reply:
Nope, I'll keep the `filter()` test since that one is a bit more exotic. 🍍